lib/commands/-command.js

```diff
@@ -6,7 +6,7 @@ module.exports = Command.extend({
   init() {
     process.env.EMBER_CLI_ELECTRON = true;
 
-    if (this._super && this._super.init) {
+    if (this._super && this._super.init && this._super.init !== this.init) {
```

Comment:
That's what I've been doing [here](https://github.com/felixrieseberg/ember-electron/blob/master/lib/commands/electron-test.js#L12), and it's working fine AFAIKT.